### PR TITLE
it: Further misc Italian translations

### PR DESCRIPTION
**Pull request by detro**
As per title.

I'm trying to get through as many "short" or "title" translations, before I engage with the big parts.

This time I haven't updated the based document, to avoid any spurious change caused by an update in the source `.po` file.

### CODE REVIEW

**Comment by mgeisler** [comment]
> This time I haven't updated the based document, to avoid any spurious change caused by an update in the source `.po` file.

Grazie! The diff looks great from that point of view 🙂

File: po/it.po [hunk 1]
```diff
@@ -824,219 +824,176 @@ msgid "Scoped Threads"
...
 #: src/SUMMARY.md:274 src/SUMMARY.md:294
-#, fuzzy
 msgid "Dining Philosophers"
 msgstr "Filosofi a tavola"
 
 #: src/SUMMARY.md:275
-#, fuzzy
 msgid "Multi-threaded Link Checker"
-msgstr "Verifica link multi-thread"
+msgstr "Verificatore di Link a thread multipli"
```
Comment (nicomazz) [inline po/it.po:872]: This sounds a bit weird, but I wouldn't know how to translate it better

File: po/it.po [hunk 2]
```diff
@@ -824,219 +824,176 @@ msgid "Scoped Threads"
...
 #: src/SUMMARY.md:277
-#, fuzzy
 msgid "Concurrency: Afternoon"
 msgstr "Giorno 1: Pomeriggio"
 
 #: src/SUMMARY.md:279
-#, fuzzy
 msgid "Async Basics"
-msgstr "Nozioni di base sull'asincronismo"
+msgstr "Nozioni di base su asincronia (Async)"
```
Comment (nicomazz) [inline po/it.po:880]: Optional nit: Not sure how to translate it better, but "asincronia" sounds pretty bad. Maybe "Async 101"?

**Review by nicomazz — APPROVED** [review]
I have to say that some automatic translations were more fun than expected. Feel free to ignore the nits i've added.

File: po/it.po (hunk 1 above)
Comment (detro) [reply po/it.po:872]: I know. But I couldn't come up with anything better.

**Comment by detro** [comment]
I pushed changes to the branch (attempts to address what @nicomazz highlighted) but... https://www.githubstatus.com/incidents/1g1gkh0qpyvs

I was really confused when I pushed but the PR wasn't changing: my luck I guess!

**Review by NoahDragon — APPROVED** [review]
Changes in `po/zh-CN.po` LGTM

**Pull request merged**